the-collab-lab/tcl-23-smart-shopping-list
# Add Icons From heroicons

## Description

This code adds the `ChevronRightIcon`, `TrashBinIcon` and `XIcon` to the necessary locations in the app based on the Figma Smart Shopping List wireframes.

It was realized that adding the clock, clipboard, and pencil icons was beneficial when the NavBar branch was being worked on, so those were not added here. Also, the `TrashBinIcon` is white in this version simply for visibility at this time, but should be changed to black when incorporated with main. 

## Related Issue

closes #34  


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|   ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### After

Added Trash Bin and X for `List` view

<img width="377" alt="trashBinIconAdded" src="https://user-images.githubusercontent.com/38255956/119733892-c6a74480-be3f-11eb-8811-62301d820d7b.png">


Added Chevron Right for `Home` view

<img width="622" alt="chevronRightIconAdded" src="https://user-images.githubusercontent.com/38255956/119733897-c73fdb00-be3f-11eb-80fe-3742e47f36c5.png">



## Testing Steps / QA Criteria

1. Navigate to the branch `add-icons`.
2. `npm install`.
3. Clear any tokens from local storage.
4. Navigate to the home screen, `localhost:3000`.
5. Click on button to create a new list.
6. Start creating a new list. Be sure to search your list for items and use the "X" to clear the input field, and to delete items from your list using the trash bin. 
7. Save your three word token.
8. Clear tokens from local storage.
9. Navigate again to the home screen, `localhost:3000`.
10. Enter your token, and click the chevron right icon to submit your token.
11. Verify that you are seeing the list you had created.
